cmd/analyse.go

```diff
@@ -65,13 +67,19 @@ func analyseResults(cmd *cobra.Command, ts *suite.TestSuite, results []result.Ne
 
 	log.Println("")
 
-	// nolint
-	op, _ := cmd.Flags().GetString("operation")
-	// nolint
-	hostname, _ := cmd.Flags().GetString("hostname")
+	//nolint
+	//op, _ := cmd.Flags().GetString("operation")
+	//nolint
+	//hostname, _ := cmd.Flags().GetString("hostname")
+	op := ""
```

Comment:
Still not set these.

Look at whats being deleted, why are you deleting these values?

Reply:
var mockCmd *cobra.Command = &cobra.Command{}